hack/tools/embed/main.go

```diff
@@ -0,0 +1,80 @@
+/*
```

Comment:
What is this utility and how is it used?

Reply:
Tis gone.